feat(themes): theme + theme starter READMEs (#15339)
gatsbyjs/gatsby: themes/gatsby-starter-theme/README.md

  </a>
</p>
<h1 align="center">
  Starter for the official Gatsby blog and notes themes.
</h1>

Quickly get started using the Gatsby blog theme, and notes theme, together! This starter creates a new Gatsby site that is preconfigured to work with the [blog theme](https://www.npmjs.com/package/gatsby-theme-blog) and the [notes theme](https://www.npmjs.com/package/gatsby-theme-notes), in harmony.

## 🚀 Quick start

1.  **Create a Gatsby site.**

    Use the Gatsby CLI to create a new site, specifying this theme starter.

    ```sh
    # create a new Gatsby site using this theme starter
    gatsby new my-themed-site https://github.com/gatsbyjs/gatsby-starter-theme
    ```

2.  **Start developing.**

    Navigate into your new site’s directory and start it up.

    ```sh
    cd my-themed-site/
    gatsby develop
    ```

3.  **Open the code and start customizing!**

    Your site is now running at `http://localhost:8000`!

    To get started, check out the guide to [getting started with using multiple themes](http://gatsbyjs.org/docs/themes/using-a-gatsby-theme), or the [general themes docs](http://gatsbyjs.org/docs/themes).

## 🧐 What's inside?

> A quick look at the top-level files and directories you'll see in a site created usingthise theme starter.

Reviewer: ```suggestion
Here are the top-level files and directories you'll see in a site created using this theme starter.
```

gatsby-starter-theme
├── content
│ ├── assets
│ │ └── avatar.jpg
│ ├── notes
│ │ ├── example-dir
│ │ │ └── hi.mdx
│ │ └── hello.mdx
│ └── posts
│ ├── hello-world.mdx
│ ├── my-second-post.mdx
│ └── new-beginnings.mdx
├── src
│ ├── gatsby-theme-blog
│ │ └── components
│ │ └── header.js
│ └── gatsby-theme-notes
│ └── components
│ └── layout.js
├── .gitignore
├── .prettierrc
├── gatsby-config.js
├── LICENSE
└── package.json

1.  **`/content`**: A content folder holding assets that the blog and notes themes expect to exist. Check out the README for each theme to learn more about the demo content.

2.  **`/src`**: You will probably want to customize your site to personalize it. The files under `/src/gatsby-theme-blog` and `/src/gatsby-theme-notes` _shadow_, or override the files of the same name in their respective packages. To learn more about this, check out the [guide to getting started with using the blog theme starter](http://gatsbyjs.org/docs/themes/using-a-gatsby-theme).

3.  **`.gitignore`**: This file tells git which files it should not track / not maintain a version history for.

> 4.  **`.prettierrc`**: This file tells git which files it should not track / not maintain a version history for.

Reviewer: ```suggestion
4.  **`.prettierrc`**: This file tells [Prettier](https://prettier.io/) which configuration it should use to lint files.
```
